File: app/src/main/java/com/example/dcct/model/internet/RegisterUserEntity.java
package com.example.dcct.model.internet;

public class RegisterUserEntity {
    private String nickname;
    private String email;
    private String password;

    @Override
    public String toString() {
        return "RegisterUserEntity{" +
                "nickname='" + nickname + '\'' +
                ", email='" + email + '\'' +
                ", password='" + password + '\'' +
                '}';
    }

    public RegisterUserEntity(String nickname, String email, String password) {
        this.nickname = nickname;
        this.email = email;
        this.password = password;
    }

    public String getNickname() {
        return nickname;
    }

    public void setNickname(String nickname) {
        this.nickname = nickname;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: app/src/main/java/com/example/dcct/bean/RegisterUserEntity.kt
package com.example.dcct.bean

class RegisterUserEntity(var nickname: String, var email: String, var password: String) {
    override fun toString(): String {
        return "RegisterUserEntity{" +
                "nickname='" + nickname + '\'' +
                ", email='" + email + '\'' +
                ", password='" + password + '\'' +
                '}'
    }

}